packages/truffle-box/lib/config.js

```diff
@@ -19,14 +17,14 @@ function setDefaults(config) {
 }
 
 function read(path) {
-  return fs
+  return fse
     .readFile(path)
     .catch(() => "{}")
```

Comment:
Kind of a weird thing to return, huh?

Reply:
Yeah, taking a second look at this, it looks like this is a fail-safe to protect truffle box writers/developers from themselves. i.e. cases where they botched the `truffle-box` or `truffle-init` json file. At this point in the flow, we've already identified the repo being unboxed is a truffle box (existence of `truffle-box.json`).